### PR TITLE
Object detection 

### PR DESCRIPTION
**Pull request by rabah-khalek**
Support for object detection models/datasets

### CODE REVIEW

File: examples/object_detection/racoon_detection/.DS_Store
Comment (Inokinoki) [inline examples/object_detection/racoon_detection/.DS_Store:1]: Let's remove the `.DS_Store`.

File: examples/object_detection/racoon_detection/Racoon Images/images/raccoon-84.jpg
Comment (Inokinoki) [inline examples/object_detection/racoon_detection/Racoon Images/images/raccoon-84.jpg:1]: Are these images cherry-picked to have only one raccoon?

File: giskard_vision/object_detection/models/wrappers.py [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+            targets (list of dicts, optional): List of target dicts for training.
+
+        Returns:
+            dict or list[dict]: The model output, either losses during training or detections during evaluation.
+        """
+        self.detector.eval()
+        return self.detector(imgs)
+
+
+class FasterRCNNBase(ModelBase):
```
Comment (Inokinoki) [inline giskard_vision/object_detection/models/wrappers.py:67]: To be removed?

**Review by Inokinoki — COMMENTED** [review]
I think most of the files are reviewed in Benoit's branch, except the racoon model.
LGTM at the moment

File: giskard_vision/core/dataloaders/base.py [hunk 2]
```diff
@@ -4,8 +4,19 @@
 
 import numpy as np
 
+from giskard_vision.core.detectors.base import IssueGroup
+
 from ..types import TypesBase
 
+EthicalIssueMeta = IssueGroup(
```
Comment (Inokinoki) [inline giskard_vision/core/dataloaders/base.py:11]: I also moved them somewhere in https://github.com/Giskard-AI/giskard-vision/pull/49

Let's merge your PR first

**Review by Inokinoki — COMMENTED** [review]
LGTM

File: giskard_vision/core/dataloaders/base.py (hunk 2 above)
Comment (rabah-khalek) [reply giskard_vision/core/dataloaders/base.py:11]: ok

File: giskard_vision/object_detection/models/wrappers.py (hunk 1 above)
Comment (rabah-khalek) [reply giskard_vision/object_detection/models/wrappers.py]: done

**Pull request merged**